lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_object.rb

```diff
@@ -161,11 +161,7 @@ def fetch_class(fqname = @class_fqname)
       Benchmark.current_realtime[:fetch_class_count] += 1
       Benchmark.realtime_block(:fetch_class_time) do
         @workspace.datastore(fqname.downcase.to_sym, :class) do
-          ns, name = MiqAeClass.parse_fqname(fqname)
-          ns = fetch_namespace(ns)
-          return nil if ns.nil?
-
-          ns.ae_classes.detect { |c| name.casecmp(c.name).zero? }
+          MiqAeClass.lookup_by_fqname(fqname)
         end
       end.first
```

Comment:
It looks like `.first` would be called on the result of line 164 `MiqAeClass.lookup_by_fqname(fqname)`?  

If so, would be nicer to moved that method call to the `lookup_by_fqname` line.